<launch>
  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix" value="" />
  <arg     if="$(arg debug)" name="launch_prefix" value="gdb -ex run --args" />

  <arg name="head" default="/" />

  <group ns="$(arg head)">
    <node name="gaze_selection" pkg="gaze_selection" type="gaze_selection_node" respawn="false" output="screen" launch-prefix="$(arg launch_prefix)">
      <param name="head_" value="$(arg head)"/>
    </node>
  </group>
</launch>


